config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml

```diff
@@ -0,0 +1,38 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-bazel-job
+      decorate: true
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        serviceAccountName: build-img
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.1
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                export PATH=$GOPATH/bin:$PATH
+                curl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/`go env GOARCH`/kubectl" -o /usr/local/bin/kubectl
+                chmod +x /usr/local/bin/kubectl
+                export GO111MODULE=on
+                export PATH=$PATH:/usr/local/bin
+                export KUBECONFIG=/etc/kubeconfig/config
+                pushd $GOPATH/src/github.com/vmware-tanzu/buildkit-cli-for-kubectl
+                make build install
+                popd
+                cd images/bazel
+                kubectl build --push --registry-secret quay-powercloud-regcred --namespace image-builder -t quay.io/powercloud/bazel:latest -f Dockerfile ./
```

Comment:
if you look at here: https://github.com/ppc64le-cloud/test-infra/blob/master/images/bazel/Makefile#L10, we are basically building 3 different version of it, lets build all those combinations instead of `:latest` tag